src/sentry/eventstream/snuba.py

```diff
@@ -410,7 +410,9 @@ def _send(
                 headers={f"X-Sentry-{k}": v for k, v in headers.items()},
             )
             if resp.status != 200:
-                raise snuba.SnubaError("HTTP %s response from Snuba!" % resp.status)
+                raise snuba.SnubaError(
+                    f"HTTP {resp.status} response from Snuba! {json.loads(resp.data)}"
```

Comment:
in case of larger response payloads you get slightly better formatting, trimming and datascrubbing if you do something like `sentry_sdk.set_extra("snuba-response", json.loads(resp.data))` rather than stringifying the payload back into the error message

Reply:
Since this is a dev only case I think the current implementation is preferable. I don't care too much if this gets to Sentry, and I *think* that using `set_extra` here won't include the response in the console?